frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -12,16 +12,19 @@ export default function Navbar() {
     async function handleLogout(event) {
         event.preventDefault();
         try {
-            const response = await axios.post("http://localhost:3000/users/logout", null, {
-                withCredentials: true
-            });
+            const response = await axios.post(
+                "http://localhost:3000/users/logout",
+                null,
+                {
+                    withCredentials: true
+                }
+            );
```

Comment:
Was this the result of running prettier?

Reply:
Yeah, sometimes prettier makes my code look weird.

Reply:
In prettier's defense I will say that in my opinion, the code looks better after it runs.